### DISCUSSION TITLE
Strange issue with GPU loading large bitmaps - Starling, Windows

### DISCUSSION THREAD

**Discussion by PippoApps** [airsdk/Adobe-Runtime-Support#3524] [Q&A]
I am working with some large bitmaps loaded in Starling (some are 6000x4096).
Everything used to work perfectly, and still works perfectly on Mac.
On Windows 10 though, when I load a large bitmap, the previously loaded one just disappears. No error is triggered, VRam is not freed, it just bum, disappears.
Same project works perfectly on Mac, and used to work on Windows as well on the same machine...
It's not a VRam issue, I use an RTX 3090 with 24Gb Vram.
This is driving me nuts... any suggestion? it used to work until a few weeks ago...

**Comment by ajwfrost** [comment]
Did you change any version of the AIR SDK? 
And would it be possible to check with advanced telemetry using Scout to see if the runtime is reporting a change in bitmap data usage or in memory clean-up etc?
thanks

**Comment by PippoApps** [comment]
Thank you Andrew.
Everything was in place, I solved by rolling back to Nvidia drivers from a few weeks ago.
So far it works :)
Cheers
F
